src/content/docs/guides/create/index.mdx

```diff
@@ -27,33 +27,33 @@ To get started using `create-tauri-app` run one of the below commands in the fol
 <Tabs>
 <TabItem label="Bash">
 ```sh
-sh <(curl https://create.tauri.app/sh)
+sh <(curl https://create.tauri.app/sh) tauri-app --alpha
```

Comment:
```suggestion
sh <(curl https://create.tauri.app/sh) --alpha
```

Just this works for me too, can you confirm that it does for you as well?

Reply:
Yeah right, it works without adding the project name in the args. I don't know what I have missed previously. Thanks for pushing the patch for this